### ISSUE TITLE
github.com/pilebones/go-udev is GPL3.0 Licensed Which Prohibits Static Linking

### ISSUE THREAD

**Issue by dpeckett** [pojntfx/go-nbd#2]
Probably should use a more liberally licensed library.

**Comment by pojntfx** [comment]
Yeah, that's def. a good point. There isn't really a good alternative available, and it is only used for a very simple purpose (to detect that a device is ready), and a fallback is available (and implemented in the form of the poll option). I'm almost thinking of just re-implementing that specific part myself ...
